com.github.Murmele.Gittyup.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id": "com.github.Murmele.Gittyup",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "Gittyup",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=network",
+        "--share=ipc",
+        "--filesystem=home",
+        "--filesystem=/tmp", /* Needed to store temporary files, which should be opened by external (non sandboxed) applications like kdiff3. Did not find another way to share a temporary file to the host (only creating a tmp folder in the home folder, I don't like) */
+        /*we use the keyring to store credentials*/
+        "--filesystem=xdg-run/keyring",
+        /*for git repos that require ssh keys*/
+        "--socket=ssh-auth",
```

Comment:
can you add `--device=dri`? Otherwise it complains about the lack of access to the gpu on wayland.

Reply:
yes I can add